src/react-components/admin/avatars.js

```diff
@@ -38,13 +45,30 @@ export const AvatarEdit = props => (
   </Edit>
 );
 
+const styles = {
+  preview: {
+    flex: 1,
+    borderRadius: 20,
+    overflow: "hidden",
+    width: 200,
+    height: 200 * (16 / 9)
+  }
+};
+
+const Preview = withStyles(styles)(({ record, classes }) => (
+  <AvatarPreview
+    className={classes.preview}
+    avatarGltfUrl={getReticulumFetchUrl(`/api/v1/avatars/${record.avatar_sid}/avatar.gltf?v=${record.updated_at}`)}
```

Comment:
maybe we should DRY up this path generation?

Reply:
Yeah I mean it should technically hit the avatar endpoint and et it from the response... But I didn't want to add another asnyc step here.